package/boot/kexec-tools/Makefile

```diff
@@ -93,10 +93,6 @@ define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) DESTDIR="$(PKG_INSTALL_DIR)" all install
 endef
 
-define Package/kexec-tools/install
-	:
-endef
```

Comment:
I think it will not If there is no `install`section no ipkg will be generated.
After this change I can select:
````
$ grep kexec .config
CONFIG_PACKAGE_kexec=y
CONFIG_PACKAGE_kexec-tools=y
````

but only kexec is generated:
````
$ ls -al build_dir/target-mips_24kc_musl/kexec-tools-2.0.26/ipkg-mips_24kc/
total 12
drwxr-xr-x  3 hauke hauke 4096 Feb 25 14:01 .
drwxr-xr-x 16 hauke hauke 4096 Feb 25 14:01 ..
drwxr-xr-x  5 hauke hauke 4096 Feb 25 14:01 kexec
````
This find nothing: `$ find bin/ -name "*kexec-tools*"`

`kexec-tools` was changed to an empta package a long time ago, see 5e4bb476c0cc46dedc020a802f045d479e483857. I think we should remove `kexec-tools` now and maybe add a `PROVIDES:=kexec-tools` to `kexec` , then opkg should install it.

Reply:
I added back the install section

Reply:
I did what @hauke said. Now I am testing.